src/servables/ensemble/ensemble_bundle.cc

```diff
@@ -57,10 +57,9 @@ EnsembleBundle::Init(const std::string& path, const ModelConfig& config)
 Status
 EnsembleBundle::SetInferenceServer(void* inference_server)
 {
-  // [TODO] Uncommon below after DLIS-290
-  // EnsembleScheduler* scheduler =
-  //    static_cast<EnsembleScheduler*>(scheduler_.get());
-  // RETURN_IF_ERROR(scheduler->SetInferenceServer(inference_server));
+  EnsembleScheduler* scheduler =
+      static_cast<EnsembleScheduler*>(GetRawScheduler());
+  RETURN_IF_ERROR(scheduler->SetInferenceServer(inference_server));
   return InferenceBackend::SetInferenceServer(inference_server);
```

Comment:
It seems like you don't actually have to store inference_server_ in backend.h. The SetInferencServer() method can just do nothing by default.